pasta_eln/GUI/projectTreeView.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, parent:QWidget, comm:Communicate, model:QStandardItemModel):
     self.setModel(model)
     self.setHeaderHidden(True)
     self.setStyleSheet('QTreeView::branch {border-image: none;}')
+    self.setExpandsOnDoubleClick(False)
```

Comment:
WHere is this function defined?

Reply:
@jmurugan-fzj  It is a function of the parent class, inherited:
https://doc.qt.io/qtforpython-6/PySide6/QtWidgets/QTreeView.html#PySide6.QtWidgets.PySide6.QtWidgets.QTreeView.setExpandsOnDoubleClick